README.md

```diff
@@ -30,11 +30,11 @@ example tests).
 
 Citation
 --------
-If you use this software please cite: [10.1101/2020.02.03.926956](https://dx.doi.org/10.1101/2020.02.03.926956)
```

Comment:
Can we keep the original link to the preprint as WELL as the link to the new nature biotech paper.

Reply:
Resolved in 4aa2233